incubator/mariadb-cluster/templates/slave-deploy.yaml

```diff
@@ -128,8 +134,16 @@ spec:
 {{ toYaml .Values.metrics.resources | indent 10 }}
 {{- end }}
       volumes:
+      - name: config
+        configMap:
+          name: {{ template "slave.fullname" . }}
       - name: data
+      {{- if .Values.slave.persistence.enabled }}
```

Comment:
I wonder whether the persistence setting should instead be cluster wide.

Reply:
well I think for a single node mariadb we should have persistence enabled by default. For the slaves could be optional (though) as it could require a lot of volumes.